lib/fog/vsphere/requests/compute/create_resource_pool.rb

```diff
@@ -6,17 +6,19 @@ def create_resource_pool(attributes = {})
           cluster = get_raw_cluster(attributes[:cluster], attributes[:datacenter])
 
           root_resource_pool = if attributes[:root_resource_pool_name]
-                                 cluster.resourcePool.find attributes[:root_resource_pool_name]
+                                 cluster.resourcePool.find attributes[:root_resource_pool_name].gsub('/', '%2f')
```

Comment:
I guess this is the main part of the fix?
Why we do not need this in another resources?
I thought that translating this characters would be a job for the underlying communication library :thinking: